### DISCUSSION TITLE
"Stopped Session 1 of User YYY.getty.shutdown.ce....I Watch.gularly.led for an unitialized xdg_surface 0x559ced1e8b40209"?

### DISCUSSION THREAD

**Discussion by Sinaaaa** [labwc/labwc#1365] [Q&A]
Hello!

`Stopped Session 1 of User YYY.getty.shutdown.ce....I Watch.gularly.led for an unitialized xdg_surface 0x559ced1e8b40209` (I typed this manually, because could not find it in the journal, so typos are not impossible)

When I reboot or turn off my PC after running Labwc and Labwc only I -often but not always- get this red systemd error message. It has no detrimental effect as far as I can tell, but I would like to ask the community what this is. What is `Watch.gulary`, please advise!  (Google & Duck does not know)

**Comment by Narrat** [comment]
The message regarding `unitialized xdg_surface` appears for me with waybar and their tooltip message for modules. Started an issue because I didn't find this (see #1372 ).
Easier way to get the labwc output: Running a nested session. Simply execute `labwc` in  a terminal.

Regarding the second question... hard to tell. Only that is has nothing to do with labwc.
Could either be the truncated output of "I watch regularly" which should be reason for concern or something systemd timer related. But I'm half joking. But there is a better way to check this instead of eye balling and the right timing. The distro you're using seems to be using systemd. Therefore `journalctl` could be of use. In combination with the `-b` option.
Example: `journalctl -b 1` lists all messages from the previous boot. And within the log you need to look for `System is powering down.` Everything after that should contain everything what happend while powering off.

**Reply by Sinaaaa** [reply]
I could not find it in journal, so I filmed the error message with my phone & then typed it with potential typos.. 
Tried this just now -again-, but searching for "surface" "xdg_surface" "powering down" led to no results..

Anyway I tried your other suggestion, which was to run labwc nested. Then I ran pkill labwc from within that session.
This is the error message that I got in my terminal:
`[types/xdg_shell/wlr_xdg_surface.c:169] A configure is scheduled for an uninitialized xdg_surface 0x5642d30eb060`
![screenshot_2023122813:4820](https://github.com/labwc/labwc/assets/8416043/85d7cd7a-4f5a-44ae-8ad8-1f6e59b895ba)

**Reply by Narrat** [reply]
For completeness:
https://github.com/labwc/labwc/issues/1372#issuecomment-1962745862
https://github.com/labwc/labwc/pull/1558
The `uninitialized xdg_surface` was solved

**Comment by Consolatis** [comment]
These are multiple messages from different applications (labwc and possibly systemd or the kernel itself) written on top of each other in the TTY.

The messages in the screenshot in red are from labwc and about a layershell popup (e.g. from some taskbar) that couldn't correctly be clipped to a monitor region, we should likely have that fixed soon. The messages above the red ones are from libxkb when compiling the keymap. The other error messages below in the screenshot are from some applications that didn't like the compositor going away.

So all in all, nothing to worry about. If you don't want to see the messages on shutdown / reboot you can redirect stderr of labwc to a file or `/dev/null`.
